src/main/resources/db/migration/V009__Model_spec_update.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE envelopes
+  ADD COLUMN classification VARCHAR(50) NOT NULL DEFAULT 'EXCEPTION',
```

Comment:
Perhaps lowercased default?

Reply:
No, because it's hibernate managed (and it will by default use uppercase)

Reply:
imho it would be best to just have `NOT NULL` here and have the logic on what value to set if it's not in the json file in application code (more explicit than java code saving null, but what ends up in db is some magic string defined in schema. but again, just my opinion).